### PR TITLE
Added GitHubFlowVersion and GitReleaseNotes, also included tagline

### PR DESCRIPTION
**Pull request by JakeGinnivan**
I also included a tagline in preparation for #17
#17 is getting harder to do because whoever does it will have to go searching. Was wondering maybe a simple `link - tagline [tags]` format would do for now, then it can be converted easily, also makes it easier to experiment.

Can remove if it is an issue for now

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```
Comment (shiftkey) [inline index.html:1]: wat

Comment (JakeGinnivan) [inline index.html:1]: o_0

No idea..

**Comment by dahlbyk** [comment]
> Was wondering maybe a simple link - tagline [tags] format would do for now

I think I'm settling on using classes like other microformats, so if you want to get a jump start on that you could do something like this:

``` html
<li><a class="proj" href="https://github.com/JakeGinnivan/GitHubFlowVersion/issues?labels=up-for-grabs&state=open">GitHubFlowVersion</a>
    - <span class="desc">Easy SemVer for projects using GitHub flow</span> [<span class="tags">github, semver</span>]</li>
```

**Comment by JakeGinnivan** [comment]
Sounds good, will update

**Comment by dahlbyk** [comment]
Note that the project list has been moved. You can just `push --force` over your existing branch.

**Pull request closed without merging**